README.md

```diff
@@ -36,20 +36,31 @@ This module gives you the possibility to integrate OneSignal into you're Appcele
 ### Usage
 1. Register device for Push Notifications
    
-   ```
+   ```js
    	    // This registers your device automatically into OneSignal
        var onesignal = require('com.williamrijksen.onesignal');
    ```
 1. To add the possibility to target people for notifications, send a tag:
    
-   ```
+   ```js
        onesignal.sendTag({ key: 'foo', value: 'bar' });
-   ```
+   ```js
 1. Delete tag:
    
    ```
        onesignal.deleteTag({ key: 'foo' });
    ```
-   
+1. Get tags (iOS only for now):
+
+    ```
+        onesignal.getTags(function(e) {
+            if (!e.success) {
+                Ti.API.error("Error: " + e.error);
+                return
+            }
+
+            Ti.API.info(e.results);
+        });
+    ```   
```

Comment:
Is it possible to add the ` onesignal.setLogLevel(onesignal.LOG_LEVEL_DEBUG);` as well into the readme? I had a look into the cordova plugin i like that syntax. Is it possible to change that function into this?

```
setLogLevel({logLevel: onesignal.LOG_LEVEL_DEBUG, visualLevel: onesignal.LOG_LEVEL_DEBUG});
```

https://documentation.onesignal.com/docs/phonegap-sdk#section--setloglevel-